src/dos/cdrom_image.cpp

```diff
@@ -1044,8 +1044,18 @@ void CDROM_Interface_Image::CDAudioCallback(const int desired_track_frames)
 		// This particular CDDA track has come to an end, but the
 		// program has requested we continue playing for a longer
 		// period. So keep going!
-		player.cd->PlayNextAudioTrack();
-		return;
+		const auto frames_remaining = player.playedTrackFrames < player.totalTrackFrames ? player.totalTrackFrames - player.playedTrackFrames : 0;
```

Comment:
@weirddan455 
picky: Please put parenthesis around these, then Clang reformat. Will end up looking like this or something.

```cpp
const auto frames_remaining = (player.playedTrackFrames < player.totalTrackFrames)
                                  ? (player.totalTrackFrames - player.playedTrackFrames) : 0;
```

Reply:
Better yet:

```
const auto frames_remaining = std::max(player.totalTrackFrames - player.playedTrackFrames, 0);
```